Mathlib/Analysis/NormedSpace/PiLp.lean

```diff
@@ -64,8 +64,6 @@ We also set up the theory for `PseudoEMetricSpace` and `PseudoMetricSpace`.
 
 set_option linter.uppercaseLean3 false
 
-local macro_rules | `($x ^ $y) => `(HPow.hPow $x $y) -- Porting note: See issue lean4#2220
```

Comment:
Feel free to also mention in the commit message that we no longer need this line :octopus: :tada: :cartwheeling:!!